Source/Core/DolphinWX/CMakeLists.txt

```diff
@@ -101,7 +101,6 @@ endif()
 
 set(DOLPHIN_EXE ${DOLPHIN_EXE_BASE})
 
-include(FindGettext)
```

Comment:
https://github.com/rukai/dolphin/blob/dolphinQt2/CMakeLists.txt
Looks like github doesnt show it because the file is too big but It is definitely checked in.